[2025-04-11T07:06:17.752+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: proyecto_bigdata.procesar_datos scheduled__2024-04-10T00:00:00+00:00 [queued]>
[2025-04-11T07:06:17.761+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: proyecto_bigdata.procesar_datos scheduled__2024-04-10T00:00:00+00:00 [queued]>
[2025-04-11T07:06:17.762+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-11T07:06:17.776+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): procesar_datos> on 2024-04-10 00:00:00+00:00
[2025-04-11T07:06:17.783+0000] {standard_task_runner.py:60} INFO - Started process 326 to run task
[2025-04-11T07:06:17.786+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'proyecto_bigdata', 'procesar_datos', 'scheduled__2024-04-10T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/proyecto_bigdata_dag.py', '--cfg-path', '/tmp/tmp9nynabqk']
[2025-04-11T07:06:17.788+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask procesar_datos
[2025-04-11T07:06:17.843+0000] {task_command.py:423} INFO - Running <TaskInstance: proyecto_bigdata.procesar_datos scheduled__2024-04-10T00:00:00+00:00 [running]> on host 4193653a407b
[2025-04-11T07:06:17.920+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='proyecto_bigdata' AIRFLOW_CTX_TASK_ID='procesar_datos' AIRFLOW_CTX_EXECUTION_DATE='2024-04-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-10T00:00:00+00:00'
[2025-04-11T07:06:17.948+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-11T07:06:17.958+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=proyecto_bigdata, task_id=procesar_datos, execution_date=20240410T000000, start_date=20250411T070617, end_date=20250411T070617
[2025-04-11T07:06:18.001+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-11T07:06:18.025+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-11T07:52:29.651+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: proyecto_bigdata.procesar_datos scheduled__2024-04-10T00:00:00+00:00 [queued]>
[2025-04-11T07:52:29.663+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: proyecto_bigdata.procesar_datos scheduled__2024-04-10T00:00:00+00:00 [queued]>
[2025-04-11T07:52:29.664+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-11T07:52:29.678+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): procesar_datos> on 2024-04-10 00:00:00+00:00
[2025-04-11T07:52:29.686+0000] {standard_task_runner.py:60} INFO - Started process 265 to run task
[2025-04-11T07:52:29.691+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'proyecto_bigdata', 'procesar_datos', 'scheduled__2024-04-10T00:00:00+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/proyecto_bigdata_dag.py', '--cfg-path', '/tmp/tmpz9xupu4u']
[2025-04-11T07:52:29.695+0000] {standard_task_runner.py:88} INFO - Job 3: Subtask procesar_datos
[2025-04-11T07:52:29.754+0000] {task_command.py:423} INFO - Running <TaskInstance: proyecto_bigdata.procesar_datos scheduled__2024-04-10T00:00:00+00:00 [running]> on host 04c54140d555
[2025-04-11T07:52:29.831+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='proyecto_bigdata' AIRFLOW_CTX_TASK_ID='procesar_datos' AIRFLOW_CTX_EXECUTION_DATE='2024-04-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-10T00:00:00+00:00'
[2025-04-11T07:52:29.858+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-11T07:52:29.867+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=proyecto_bigdata, task_id=procesar_datos, execution_date=20240410T000000, start_date=20250411T075229, end_date=20250411T075229
[2025-04-11T07:52:29.905+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-11T07:52:29.946+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
